src/http/two/BinaryTokenizer.js

```diff
@@ -0,0 +1,148 @@
+export class InsufficientInputError extends Error {}
+export class WrongSkipError {
+    constructor(intendedSkip, actuallySkipped, ...args) {
+        this.intendedSkip = intendedSkip;
+        this.actuallySkipped = actuallySkipped;
+        this.message = args.join(" ");
+    }
+}
+
+export default class BinaryTokenizer {
+    constructor(data = "") {
+        this._data = data;
+        this._parsed = 0;
+        this._commitPoint = 0;
+    }
+
+    commit() {
+        this._commitPoint = this._parsed;
+    }
+
+    rollback() {
+        this._parsed = this._commitPoint;
+    }
+
+    binToHex(text) {
```

Comment:
Make static or [re]move. Capitalize if it becomes static or global. Avoid code duplication.